integration-cli/docker_cli_images_test.go

```diff
@@ -363,3 +363,20 @@ func (s *DockerSuite) TestImagesFormatDefaultFormat(c *check.C) {
 	out, _ = dockerCmd(c, "--config", d, "images", "-q", "myimage")
 	c.Assert(out, checker.Equals, imageID+"\n", check.Commentf("Expected to print only the image id, got %v\n", out))
 }
+
+func (s *DockerSuite) TestImageSortBy(c *check.C) {
+	dockerCmd(c, "tag", "busybox", "name1:v1")
+	dockerCmd(c, "tag", "busybox", "name1:v2")
+	dockerCmd(c, "tag", "busybox", "name0:v2")
+
+	imagesOut, _ := dockerCmd(c, "images", "--sort-by", "Repository:asc,tag:desc", "name*")
```

Comment:
Because this is the correct usage, as a compare, line 379 demonstrate invalid options.